src/requisition-full-supply/full-supply.controller.js

```diff
@@ -171,6 +171,20 @@
             return column.name === TEMPLATE_COLUMNS.SKIPPED;
         }
 
+        /**
+         * @ngdoc method
+         * @methodOf requisition-full-supply.controller:FullSupplyController
+         * @name canEditRequisition
+         *
+         * @description
+         * Determines whether the requisition is editable.
+         *
+         * @return {Boolean} true if requisition is editable
+         */
+        function canEditRequisition() {
```

Comment:
I think this is unduplicated enough... for now

the problem is the markup that needs these functions exposed — which could be replaced with direct calls to vm.requisition.$isEditable.

Main functionality is abstracted back to a single point that is easy to extend

@ngraczewski — I'd megre this, but I'd like you to agree